assets/js/components/utils/index.js

```diff
@@ -3,18 +3,24 @@
  */
 import { addQueryArgs } from '@wordpress/url';
 import apiFetch from '@wordpress/api-fetch';
-import { flatten, uniqBy } from 'lodash';
+import { flatten, uniqBy, merge } from 'lodash';
 
 export const isLargeCatalog = wc_product_block_data.isLargeCatalog || false;
 
-const getProductsRequests = ( { selected = [], search } ) => {
+const getProductsRequests = ( { selected = [], search = '', queryArgs = [] } ) => {
+	const defaultArgs = {
+		per_page: isLargeCatalog ? 100 : -1,
+		catalog_visibility: 'visible',
+		status: 'publish',
+		search,
+		orderby: 'title',
+		order: 'asc',
+	};
 	const requests = [
-		addQueryArgs( '/wc/blocks/products', {
-			per_page: isLargeCatalog ? 100 : -1,
-			catalog_visibility: 'visible',
-			status: 'publish',
-			search,
-		} ),
+		addQueryArgs(
+			'/wc/blocks/products',
+			merge( defaultArgs, queryArgs )
```

Comment:
Considering we don't need recursive merging here, I think the native rest operator would be more performant:
```suggestion
			{ ...defaultArgs, ...queryArgs }
```

Reply:
Thanks, the spread operator is new to me.